Zend/tests/readonly_props/readonly_clone_success4.phpt

```diff
@@ -4,7 +4,9 @@ Readonly property can be reset once during cloning even after a type error
 <?php
 
 class Foo {
-    public readonly int $bar;
+    public function __construct(
+        public readonly int $bar
+    ) {}
```

Comment:
Would be good if the value in `__clone` was changed to `2` to verify it actually changes.